packages/airnode-protocol/README.md

```diff
@@ -21,8 +21,13 @@ yarn run test:coverage
 yarn run test:gas
 ```
 
-Deploy the contracts on a network (`<NETWORK>` must be one of the names from `credentials.example.json`), verify the
-deployment
+### Contract deployment
+
+To deploy the contracts on a network first copy `example.env` to `.env`. Enter a mnemonic and any appropriate API keys.
+Note that the mnemonic must be present (though can be arbitrary) even if the exclusive goal is contract verification.
+The `<NETWORK>` in the below commands must correspond to a network in `TODO`. If a hardhat error occurs during
```

Comment:
What is our source of truth for network names? The file that was being referred to (`credentials.example.json`) does not exist...

Reply:
Aliases from https://github.com/api3dao/chains

Reply:
To make it easier, file names from https://github.com/api3dao/chains/tree/main/chains